src/envoy/utils/authn.cc

```diff
@@ -74,6 +80,16 @@ void Authentication::SaveAuthAttributesToStruct(
       setKeyValue(data, istio::utils::AttributeName::kRequestAuthAudiences,
                   origin.audiences(0));
     }
+    if (!origin.groups().empty()) {
```

Comment:
This is about to be removed. You need to add this logic to SaveAuthAttributesToStruct instead.

Reply:
Have discussed it offline and it is already in SaveAuthAttributesToStruct.